### PR TITLE
Revert "Build SpacemanDMM from source"

### PR DESCRIPTION
**Pull request by optimumtact**
Reverts tgstation/tgstation#49712

This should not have been merged.

Master has been failing Travis because dmdoc builds still rely on install_spaceman_dmm.sh.

Additionally, the new lints have been raising errors in VSC because a new release of the language server had not been made. While this has now been corrected by me post-hoc because someone actually complained to me directly, it should really have been brought to my attention earlier.

### CODE REVIEW

**Review by MrsTonedOne — APPROVED** [review]

File: code/modules/antagonists/abductor/equipment/abduction_gear.dm [hunk 1]
```diff
@@ -518,7 +518,7 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 		toggle_on(user)
 
 	if(iscyborg(target))
-		if(mode == BATON_STUN)
+		if(BATON_STUN)
```
Comment (ShizCalev) [inline code/modules/antagonists/abductor/equipment/abduction_gear.dm]: is this right?

File: code/game/objects/effects/portals.dm [hunk 2]
```diff
@@ -229,6 +229,7 @@
 			var/obj/projectile/P = M
 			P.ignore_source_check = TRUE
 		return TRUE
+	return FALSE
```
Comment (ShizCalev) [inline code/game/objects/effects/portals.dm]: that can't be right at all.

File: code/modules/mob/living/carbon/human/species_types/snail.dm [hunk 3]
```diff
@@ -41,7 +41,7 @@
 	var/obj/item/storage/backpack/bag = C.get_item_by_slot(ITEM_SLOT_BACK)
 	if(istype(bag, /obj/item/storage/backpack/snail))
 		bag.emptyStorage()
-		C.temporarilyRemoveItemFromInventory(bag, TRUE)
+		C.doUnEquip(bag, TRUE, no_move = TRUE)
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/species_types/snail.dm]: .

File: code/game/machinery/doors/firedoor.dm [hunk 4]
```diff
@@ -466,7 +466,7 @@
 			constructionStep = CONSTRUCTION_GUTTED
 			update_icon()
 			return TRUE
-		if(RCD_DECONSTRUCT)
+		else if(RCD_DECONSTRUCT)
```
Comment (ShizCalev) [inline code/game/machinery/doors/firedoor.dm]: .

File: code/datums/diseases/gastrolisis.dm [hunk 5]
```diff
@@ -82,5 +82,5 @@
 		var/obj/item/storage/backpack/bag = H.get_item_by_slot(ITEM_SLOT_BACK)
 		if(istype(bag, /obj/item/storage/backpack/snail))
 			bag.emptyStorage()
-			H.temporarilyRemoveItemFromInventory(bag, TRUE)
+			H.doUnEquip(bag, TRUE, no_move = TRUE)
```
Comment (ShizCalev) [inline code/datums/diseases/gastrolisis.dm]: .

**Comment by spookydonut** [comment]
```
executable: c:\Users\xxx\.vscode\extensions\platymuus.dm-langclient-0.1.6\bin\dm-langserver-x64-win32.exe
commit: 3cd3c402af04e6deedc9149caf5c5b0dfd44ad3b
build date: 2020-03-08UTC
```
https://github.com/SpaceManiac/SpacemanDMM/commit/3cd3c402af04e6deedc9149caf5c5b0dfd44ad3b

**Comment by optimumtact** [comment]
no

**Comment by spookydonut** [comment]
langserver has been updated ages ago, i only removed install spacemandmm.sh because @ninjanomnom told me to, i wanted to keep it

**Comment by spookydonut** [comment]
See #49858 for why I slapped DNM on this

**Comment by optimumtact** [comment]
The maintainer of the tool has requested we undo this, you're just going to have to wait on the release cycles

**Pull request closed without merging**